File: typescript/src/n0169_majority_element.test.ts
import { expect, test } from 'bun:test';

import { majorityElement } from './n0169_majority_element';

test('majorityElement', () => {
  expect(majorityElement([3, 2, 3])).toEqual(3);
  expect(majorityElement([2, 2, 1, 1, 1, 2, 2])).toEqual(2);
});

File: rust/src/n0169_majority_element.test.rs
use crate::n0169_majority_element::majority_element;

#[test]
fn test_majority_element() {
    assert_eq!(majority_element(&[3, 2, 3]), 3);
    assert_eq!(majority_element(&[2, 2, 1, 1, 1, 2, 2]), 2);
}
